README.md

```diff
@@ -0,0 +1,192 @@
+
+If you still need help, join us on the [Discord server]( https://discord.gg/gnpVs5m ), where you'll find Channels for support,
+add-on plugins, [SponsorPlugins](https://github.com/LlmDl/SponsorPlugins#readme), discuss plugin-development and just regular old discussion of the plugin.
+
+**Note: If you are not a new Towny admin, it is expected that you [become a Sponsor](https://github.com/sponsors/LlmDl) if you want hands on support from LlmDl.**
+
+___
+
+### Contributing
+If you'd like to contribute to the Towny code, see the [CONTRIBUTING.md](https://github.com/LlmDl/Towny/blob/master/.github/CONTRIBUTING.MD) over on Towny.
+
+___
+
+### Translations
+
+TownyChat does not offer its own localization. Instead, this is partially handled within Towny.
+
+If you'd like to help translating Towny into the available languages or add entirely new languages, [we're on Crowdin!](https://crowdin.com/project/townyadvanced)
+
+[![Crowdin](https://badges.crowdin.net/townyadvanced/localized.svg)](https://crowdin.com/project/townyadvanced)
+
+___
+
+### Licensing
+Towny & TownyChat are licensed under the [Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported (CC BY-NC-ND 3.0) License ](https://creativecommons.org/licenses/by-nc-nd/3.0/)
+You can view a repo-local copy of this license at [LICENSE.md](./LICENSE.md).
+
+We don't object to you making your own personal-use forks and builds, but we do object to people being selfish, which is why we specify No Derivative Works.
+If you want to modify the code to add a nice feature or two, the least you can do is ask and submit a Pull Request to allow everyone to benefit from it.
+
+___
+
+### Hooking directly with TownyChat
+[![](https://jitpack.io/v/TownyAdvanced/TownyChat.svg)](https://jitpack.io/#TownyAdvanced/TownyChat)
+
+You can hook directly into TownyChat, if you wish. This is however, unsupported.
+Builds of TownyChat are not currently published to the Towny repository hosted at https://repo.glaremasters.me/repository/towny/.
+You will need to grab it from Jitpack, or install it to your local Maven repository.
+
+We encourage developers who wish to use the Towny API in their plugins to read
+[the following instructions on adding Towny via a Maven pom.xml,](https://github.com/TownyAdvanced/Towny/wiki/TownyAPI#getting-started-with-towny-and-your-ide)
+or read [the API guide](https://github.com/TownyAdvanced/Towny/wiki/TownyAPI) to learn about the vast API Towny has to offer.
+
+___
+
+### Building TownyChat
+If you would like to build TownyChat yourself, you can do so with [Apache Maven](https://maven.apache.org/).
+Depending on the age of the branch, you may wish to instead look at [Apache Ant](https://ant.apache.org).
+Building with Ant is outside of this brief tutorial's scope.
```

Comment:
TownyChat doesn't have any branches that require ant so it's probably not worth mentioning here